test/MFEMHesthavenComparison/TestMfemHesthavenTrace1D.cpp

```diff
@@ -0,0 +1,477 @@
+#pragma once
+
+#include "TestMfemHesthavenFunctions.h"
+#include "../TestGlobalFunctions.h"
+
+class MFEMHesthaven1DTrace : public ::testing::Test {
+protected:
+
+	void SetUp() override
+	{
+		mesh_ = Mesh::MakeCartesian1D(1);
+		fec_ = std::make_unique<DG_FECollection>(1, 1, BasisType::GaussLobatto);
+		fes_ = std::make_unique<FiniteElementSpace>(&mesh_, fec_.get());
+	}
+
+	void setFES(const int order, const int elements = 1)
+	{
+		mesh_ = Mesh::MakeCartesian1D(elements);
+		fec_ = std::make_unique<DG_FECollection>(order, 1, BasisType::GaussLobatto);
+		fes_ = std::make_unique<FiniteElementSpace>(&mesh_, fec_.get());
+	}
+
+	Mesh mesh_;
+	std::unique_ptr<FiniteElementCollection> fec_;
+	std::unique_ptr<FiniteElementSpace> fes_;
+
+	double tol_ = 1e-6;
+
+};
+
+
+TEST_F(MFEMHesthaven1DTrace, checkStrongFluxOperator)
+{
+
+	Mesh mesh = Mesh::MakeCartesian1D(1);
+	std::unique_ptr<FiniteElementCollection> fec = std::make_unique<DG_FECollection>(2, 1, BasisType::GaussLobatto);
+	std::unique_ptr<FiniteElementSpace> fes = std::make_unique<FiniteElementSpace>(&mesh, fec.get());
+
+	auto mInv = buildInverseMassMatrixEigen(fes);
+	std::cout << mInv << std::endl;
```

Comment:
This is not an automatic test... I guess you want to see the result if they don't match. I would recommend an element by element comparison.

Reply:
Still skeleton code. WIP.